Doc/whatsnew/3.8.rst

```diff
@@ -237,6 +237,37 @@ asyncio
 
 On Windows, the default event loop is now :class:`~asyncio.ProactorEventLoop`.
 
+builtins
+--------
+
+The :func:`compile` built-in has been improved to accept the
+``ast.PyCF_ALLOW_TOP_LEVEL_AWAIT`` flag. With this new flag passed,
+:func:`compile` will allow top-level ``await``, ``async for`` and ``async with``
+constructs that are usually considered invalid syntax. Asynchronous code object
+marked with the ``CO_COROUTINE`` flag may then be returned. Those code-objects
+can be used in conjunction with ``await eval(...)`` (instead of ``exec``) to for
```

Comment:
Should "to" be "too" as in "Code objects can be used in conjunction with ``await eval(...)`` (instead of ``exec``) too. For example run async snippets in doctest, or build an async-repl."

Reply:
I don't believe so; it's "They can be use to run", not "they can be used also". But it does not matter as per Yuri comment, I removed this and the example.